numpy_basics/numpy_basics.ipynb

```diff
@@ -0,0 +1,2780 @@
+{
```

Comment:
<p>[inclusive, exclusive] is a little bit misleading. </p><p> </p><p>[i,j] -&gt; [x[:i], x[i:j], x[j:]]  would be more accurate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/syncdoth/ML_Study_2020/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>